apps/entry/tests/test_apis.py

```diff
@@ -682,6 +682,57 @@ def test_authoring_organization_filter(self):
 
     # TODO: test export data and filter data apis
 
+    def test_entry_id_filter(self):
+        user = self.create_user()
+        project = self.create_project()
+        project.add_member(user)
+        entry1 = self.create_entry(project=project)
+        entry2 = self.create_entry(project=project)
+        entry3 = self.create_entry()
+        filters = {
+            'entry_ids': [entry1.pk, entry3.pk],
+        }
+        url = '/api/v1/entries/filter/'
+        params = {
+            'filters': [[k, v] for k, v in filters.items()]
+        }
+
+        self.authenticate(user)
+        response = self.client.post(url, params)
+
+        self.assert_200(response)
+        assert len(response.json()['results']) == 1  # only the entry of project that user is member
+
+        # try filtering out the entries that the user is not member of
+        filters = {
+            'entry_ids': [entry1.pk, entry2.pk, entry3.pk],
+        }
+        url = '/api/v1/entries/filter/'
+        params = {
+            'filters': [[k, v] for k, v in filters.items()]
+        }
+
+        self.authenticate(user)
+        response = self.client.post(url, params)
+
+        self.assert_200(response)
+        assert len(response.json()['results']) == 2  # only the entry of project that user is member
+
+        # try without authenticating the user
```

Comment:
`self.authenticate()` doesn't log out the user. So this comment is wrong.

Let's update this saying, try with default user which is a member in all projects.